python.wake

```diff
@@ -82,13 +97,14 @@ target installPipenvEnv pipDir =
 
   def visible = installPipenvScript, pipFile, lock, Nil
 
+  # Create a plan to run the script and create a virtual environment.
   makePlan args visible
   | setPlanResources ("python/python/3.7.1", Nil)
-  | runJob
-  | getJobOutputs
-
 
+  # Disable Fuse since it is incompatible with pipenv.
+  | setPlanLocalOnly True
```

Comment:
I'm worried about this line because setting a plan to be local-only has a very powerful and contagious effect, preventing other commands from running correctly. I feel the real problem is with how the Wake preload runner works, since this is a "leaky" part of the abstraction.

Can we back out of this half of the change and restrict this PR to only updating the `npm` command? We can file a GitHub issue on this repository to track and discuss the other issue.